<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QMainWindow" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Poze/Resources/blue.jpg);</string>
   </property>
   <widget class="QPushButton" name="StartGame">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>450</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:transparent;
background-color: qlineargradient(spread:pad, x1:0.443, y1:0, x2:0.465909, y2:1, stop:0 rgba(81, 85, 97, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:transparent;
background-color: qlineargradient(spread:pad, x1:0.443, y1:0, x2:0.465909, y2:1, stop:0 rgba(81, 85, 97, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:transparent;
background-color: qlineargradient(spread:pad, x1:0.443, y1:0, x2:0.465909, y2:1, stop:0 rgba(81, 85, 97, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:transparent;
background-color: qlineargradient(spread:pad, x1:0.443, y1:0, x2:0.465909, y2:1, stop:0 rgba(81, 85, 97, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="Player4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:transparent;
background-color: qlineargradient(spread:pad, x1:0.443, y1:0, x2:0.465909, y2:1, stop:0 rgba(81, 85, 97, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Registration.qrc"/>
 </resources>
 <connections/>
</ui>
